alacritty/src/main.rs

```diff
@@ -85,18 +85,10 @@ fn main() {
         .expect("Unable to initialize logger");
 
     // Load configuration file
-    // If the file is a command line argument, we won't write a generated default file
-    let config_path = options
-        .config_path()
-        .or_else(config::installed_config)
-        .or_else(|| config::write_defaults().ok())
-        .map(|path| path.to_path_buf());
-    let config = if let Some(path) = config_path {
-        config::load_from(path)
-    } else {
-        error!("Unable to write the default config");
-        Config::default()
-    };
+    let config_path =
+        options.config_path().or_else(config::installed_config).map(|path| path.to_path_buf());
```

Comment:
Do we need to convert to `PathBuf` here? I think opening files should work with `Path` too, but I'm not entirely certain. No need to convert unless we need it of course.

Reply:
@kchibisov This is not resolved.